programs/ziti-edge-tunnel/package/CPackGenConfig.cmake

```diff
@@ -29,6 +29,6 @@ if(CPACK_GENERATOR MATCHES "RPM")
     endif(CPACK_GENERATOR MATCHES "RPM")
 
 if(CPACK_GENERATOR MATCHES "DEB")
-    set(CPACK_DEBIAN_PACKAGE_DEPENDS "iproute2, sed, systemd, libatomic1")
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "iproute2, sed, systemd, libatomic1, openssl")
```

Comment:
just need the library package `libssl3`

Reply:
I did find `openssl-libs` for the RPM, but for DEB the `openssl` package has no deps, and so it appears to be the correct package for the runtime. Buildtime uses `libss-dev`.